server/src/namesController.js

```diff
@@ -0,0 +1,79 @@
+const nameQueries = require('./nameQueries');
```

Comment:
Conversely to `nameQueries`, this module has a focused responsibility of parsing HTTP requests and constructing responses. It interacts with the queries module and uses query results to determine the appropriate response.